app/styles/app.scss

```diff
@@ -556,12 +558,13 @@ hr {
             opacity: 1;
         }
     }
-    .row {
-        margin: 0;
-    }
-    label, input, textarea {
-        display: block;
-    }
+    // THESE STYLES ARE INTERFERING WITH MY STYLES. 
```

Comment:
Agreed. As long as there's a path forward, sounds good by me.

In other addons, we try to namespace our styles to make them less likely to collide with other components. This will be an area to ticket for future improvements.